spyder/plugins/editor/widgets/codeeditor.py

```diff
@@ -2179,7 +2207,6 @@ def fix_indent(self, *args, **kwargs):
             return self.fix_indent_smart(*args, **kwargs)
         else:
             return self.simple_indentation(*args, **kwargs)
-        self.document_did_change()

```

Comment:
Why did you remove this? This prevents linting to be updated when `fix_indent` is called.

Reply:
Well it is after the return of the function, so it won't get executed either way. Should the function be changed?

Reply:
Right, I didn't take into account. Then I think it should be added to `fix_indent_smart` and `simple_indentation`.

Reply:
I changed it so document_did_change is called